roles/openshift_version/tasks/main.yml

```diff
@@ -84,115 +84,121 @@
   - openshift_version is not defined
   - openshift_protect_installed_version | bool
 
-- name: Set openshift_version for rpm installation
-  include: set_version_rpm.yml
-  when: not is_containerized | bool
-
-- name: Set openshift_version for containerized installation
-  include: set_version_containerized.yml
-  when: is_containerized | bool
-
-- block:
-  - name: Get available {{ openshift.common.service_type}} version
-    repoquery:
-      name: "{{ openshift.common.service_type}}"
-      ignore_excluders: true
-    register: rpm_results
-  - fail:
-      msg: "Package {{ openshift.common.service_type}} not found"
-    when: not rpm_results.results.package_found
-  - set_fact:
-      openshift_rpm_version: "{{ rpm_results.results.versions.available_versions.0 | default('0.0', True) }}"
-  - name: Fail if rpm version and docker image version are different
-    fail:
-      msg: "OCP rpm version {{ openshift_rpm_version }} is different from OCP image version {{ openshift_version }}"
-    # Both versions have the same string representation
+# The rest of these tasks should execute only on
+# non nfs and lb hosts as these hosts may or may not
+# have subscriptions set up
+- when:
+  - inventory_hostname not in hostvars[groups.oo_nfs_to_config]
```

Comment:
maybe like this from roles/etcd_migrate/tasks/check_cluster_status.yml
  when: inventory_hostname in groups.oo_etcd_to_config[0]